app/src/main/java/org/wikipedia/imagesearch/Encoder.java

```diff
@@ -0,0 +1,69 @@
+package org.wikipedia.imagesearch;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.util.Base64;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Created by mnhn3 on 2018-04-12.
+ */
+
+public class Encoder {
+
+    public String encodeFileToBase64Binary(File file) {
```

Comment:
What's the purpose of this method?

Reply:
Although this method isn't explicitly used, it could be useful in the future as it allows for various input types if `Context` and `Uri` aren't available, for whatever reason.